Formula/a/arjun.rb

```diff
@@ -1,9 +1,12 @@
 class Arjun < Formula
+  include Language::Python::Virtualenv
+
   desc "HTTP parameter discovery suite"
   homepage "https://github.com/s0md3v/Arjun"
   url "https://files.pythonhosted.org/packages/83/2d/e521035e38c81c9d7f4aa02a287dddeb163ad51ebca28bef7563fc503c07/arjun-2.2.2.tar.gz"
   sha256 "3b2235144e91466b14474ab0cad1bcff6fb1313edb943a690c64ed0ff995cc46"
   license "AGPL-3.0-only"
+  revision 1
```

Comment:
I put in this rev bump because I initially I thought it potentially worthwhile to force rebuilding bottles, but since the package isn't broken like [aws-sam-cli is](https://github.com/Homebrew/homebrew-core/pull/162634), maybe we can drop this.

Reply:
Yeah, we should use revisions very sparingly. Better to remove it than force an update on every user.